jumpcloud-ADMU/Powershell/Private/Permissions/Set-RegPermission.ps1

```diff
@@ -61,7 +59,7 @@ function Set-RegPermission {
 
     # Step 1: Grant target user full control inheritance on root folder
     Write-ToLog "Granting permissions to: $TargetAccountIcacls" -Level Verbose -Step "Set-RegPermission"
-    $icaclsGrantResult = icacls $FilePath /grant "${TargetAccountIcacls}:(OI)(CI)F" /T /C /Q
+    $icaclsGrantResult = icacls $FilePath /grant "${TargetAccountIcacls}:(OI)(CI)F" /T /C /Q > $ntfsPermissionLogPath 2>&1
```

Comment:
Here I'm writing output from icalcs to a log file that would otherwise be lost to the host stream